src/classes/REL_Relationships_TEST.cls

```diff
@@ -160,26 +154,26 @@ public class REL_Relationships_TEST {
     public static void testOrphanedRelDelete() {
         setupRelationshipTestData(null);
 
-        Relationship__c crel = new Relationship__c(contact__c = c1.id, Relatedcontact__c = c2.id, Type__c='Buddies');
+        Relationship__c crel = new Relationship__c(Contact__c = c1.Id, RelatedContact__c = c2.Id, Type__c='Buddies');
         insert crel;
 
         // check that rel gets created
-        Relationship__c crel_original = [select id, Description__c,Type__c,reciprocalrelationship__c
-                                        from Relationship__c where id=:crel.Id];
+        Relationship__c crel_original = [SELECT Id, Description__c,Type__c,ReciprocalRelationship__c
+                                        FROM Relationship__c WHERE Id=:crel.Id];
         //relationship should be set right
         System.assertEquals(crel_original.Type__c, 'Buddies');
 
         // check for mirror rel
-        Relationship__c crel_mirror = [select id, Description__c, reciprocalrelationship__c, Type__c
-                                        from Relationship__c where reciprocalrelationship__c=:crel_original.Id];
+        Relationship__c crel_mirror = [SELECT Id, Description__c, ReciprocalRelationship__c, Type__c
+                                        FROM Relationship__c WHERE ReciprocalRelationship__c=:crel_original.Id];
```

Comment:
"...WHERE ReciprocalRelationship__c = :crel_original.Id" -- just some spacing while we are in here. It's in a few spots: line 162, this one (168), 191, 196. (Might be more but those are the ones i've spotted). If ya don't mind fixing those since we are getting in the habit of fixing these types of things :)

Reply:
This one should be easy because I might be able to do a find/replace for __c=: :) (that's how I was able to do most of these...)